.github/workflows/tmt-tests.yml

```
@@ -76,24 +76,24 @@ jobs:
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
       tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*max_sst)"
-      variables: 'SOURCE_RELEASE=8.6;TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms;LEAPPDATA_BRANCH=upstream'
+      variables: 'SOURCE_RELEASE=8.6;TARGET_RELEASE=9.0;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms;LEAPPDATA_BRANCH=upstream'
       pull_request_status_name: "8.6to9.0"
     if: |
       github.event.issue.pull_request
       && ! startsWith(github.event.comment.body, '/rerun-sst')
       && contains(fromJson('["OWNER", "MEMBER", "COLLABORATOR"]'), github.event.comment.author_association)
 
-  call_workflow_tests_87to90_integration:
+  call_workflow_tests_88to92_integration:
     needs: call_workflow_copr_build
     uses: ./.github/workflows/reuse-tests-8to9.yml
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
       tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*max_sst)"
-      variables: 'SOURCE_RELEASE=8.7;TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms;LEAPPDATA_BRANCH=upstream'
-      compose: 'RHEL-8.7.0-Nightly'
-      pull_request_status_name: "8.7to9.0"
-      tmt_context: "distro=rhel-8.7"
+      variables: 'SOURCE_RELEASE=8.8;TARGET_RELEASE=9.2;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms;LEAPPDATA_BRANCH=upstream'
```

This is a beta to beta upgrade so I think we need some additional changes:
```suggestion
      variables: 'SOURCE_RELEASE=8.8;TARGET_RELEASE=9.2;RHSM_REPOS=rhel-8-for-x86_64-appstream-beta-rpms,rhel-8-for-x86_64-baseos-beta-rpms;LEAPP_DEVEL_TARGET_RELEASE=9.2;LEAPPDATA_BRANCH=upstream'
```

---

Right, thanks. Currently it doesn't matter for upstream tests as they are non-rhsm, but let's do it right.

---

@fernflower should I update that in packit for both leapp and leapp repository too?